samples/deid.js

```diff
@@ -43,7 +43,7 @@ async function deidentifyWithMask(
   // Construct deidentification request
   const item = {value: string};
   const request = {
-    parent: dlp.projectPath(callingProjectId),
+    parent: `projects/${callingProjectId}/locations/global`,
```

Comment:
@alexander-fenster should this be `.locationPath`?

Reply:
I don't believe a .locationPath template exists. If it does, where would it be defined?

Reply:
So, after talking with more people, it sounds like getting a locationPath template to appear is going to require some annotation changes in our DLP proto (what exactly we need to do is not yet clear). 

I don't think we should block this change on getting that resolved. Using a template is preferred, and I'd like to get to that state, but in the meantime I'd rather have working samples even if they involve string interpolation instead of templates.